### PR TITLE
Use BPS instead of TPS

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Replaced TPS with BPS

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by marc2332 — CHANGES_REQUESTED** [review]
Also change the `EpochReward` type as it currently uses `totalTps`

**Comment by Dr-Electron** [comment]
> Also change the EpochReward type as it currently uses totalTps

Not sure what you mean with that 😅 Where does it use totalTps?

**Comment by marc2332** [comment]
> > Also change the EpochReward type as it currently uses totalTps
> 
> Not sure what you mean with that 😅 Where does it use totalTps?

Ah yes, I just remembered that for some reason it no longer has the `totalTps` field, but it should. Let's add it as `totalBps`.

image
image

File: src/components/HomeLayout/styles.css [hunk 1]
```diff
@@ -682,3 +682,66 @@
...
+
+.mana-calculator__button {
+  background-color: var(--ifm-color-emphasis-100);
+  margin: 0px;
+  margin-top: 1rem;
+}
+
+.mana-calculator__tooltip > p {
+  margin: 0;
+}
```
Comment (marc2332) [inline src/components/HomeLayout/styles.css:747]: I think you got these duplicated, just remove them from here as I think they should be in ManaCalculator/styles.css

**Review by marc2332 — CHANGES_REQUESTED** [review]

File: src/components/HomeLayout/styles.css (hunk 1 above)
Comment (Dr-Electron) [inline src/components/HomeLayout/styles.css:747]:
```suggestion
```

**Review by marc2332 — APPROVED** [review]

**Pull request merged**